API.php

```diff
@@ -25,10 +25,10 @@
  */
 class API extends \Piwik\Plugin\API
 {
-    protected function getDataTable($name, $idSite, $period, $date, $segment, $expanded = false, $idSubtable = null)
+    protected function getDataTable($name, $idSite, $period, $date, $segment, $expanded = false, $idSubtable = null, $flat = false)
```

Comment:
For consistency reasons I guess I would prefer to use the same order of parameters as for `Archive::createDataTableFromArchive`

```suggestion
    protected function getDataTable($name, $idSite, $period, $date, $segment, $expanded = false, $flat = false, $idSubtable = null)
```